src/scripts/browserDeviceProfile.js

```diff
@@ -743,28 +743,6 @@ export function canPlaySecondaryAudio(videoTestElement) {
             }
         }
 
-        // Progressive mp4 transcoding
-        if (mp4VideoCodecs.length && videoAudioCodecs.length) {
-            profile.TranscodingProfiles.push({
-                Container: 'mp4',
-                Type: 'Video',
-                AudioCodec: videoAudioCodecs.join(','),
-                VideoCodec: mp4VideoCodecs.join(','),
-                Context: 'Streaming',
-                Protocol: 'http',
-                MaxAudioChannels: physicalAudioChannels.toString()
-            });
-        }
-
-        profile.TranscodingProfiles.push({
```

Comment:
Bill, weren't you going to use `Static=true` to download transcoded media in Expo app? AFAIK, that's what `Context: 'Static'` is used for.

If we are going to drop MP4 progressive streaming, then I need to drop the corresponding commit (with `CopyTimestamps`) in https://github.com/jellyfin/jellyfin-web/pull/4345

MKV also has progressive variant (not supported by browsers though):
https://github.com/jellyfin/jellyfin-web/blob/d8c3facf4c94a942ee626a77cf868f6eaa38b41e/src/scripts/browserDeviceProfile.js#L693-L701

Reply:
The downloads for the Expo app I just manually build a stream URL for a progressive stream. (Refs: https://github.com/jellyfin/jellyfin-expo/blob/0d3456dc4f38cc8030985d98dba7dcbf9e06f1d2/models/DownloadModel.ts#L63-L77)

We aren't talking about completely removing progressive streaming right? Just removing support from web due to seeking issues?

I do think we should consider still adding this if the transcoding profiles is empty though to avoid having no transcode profile at all... I realize that _shouldn't_ happen, but it still doesn't feel safe to me. :sweat_smile:

Reply:
Even Tizen 2.3 (webOS 1.2 probably too) [supports](https://developer.samsung.com/smarttv/develop/specifications/general-specifications.html) HLS.

`CopyTimestamps: true` might help with desyncing (similar to https://github.com/jellyfin/jellyfin-web/pull/4345), but seeking is still weird (bound to I-frames) and restarting the transcoding job.
_Btw, Nyan is the one who suggested me the `copyts` flag._

Reply:
Okay I'm convinced. Let's give it a shot. :crossed_fingers: